.circleci/config.yml

```diff
@@ -43,41 +62,33 @@ jobs:
           root: ~/project
           paths:
             - yarn
-  test:
-    <<: *defaults
-    steps:
-      - attach_workspace:
-          at: ~/project
-      - *restore_node_modules
-      - run:
-          name: Tests
-          command: |
-            node -v
-            if [ "$CIRCLE_BRANCH" == 'master' ]; then
-              ./scripts/set-dev-version.js
-            fi;
-            # Limit maxWorkers to 3 to avoid OOM on CircleCI
-            yarn test-ci --maxWorkers 3
-            yarn check-lockfile
   lint:
     <<: *defaults
     steps:
-      - attach_workspace:
-          at: ~/project
-      - *restore_node_modules
+      - *attach_workspace
+      - *restore_cache
       - run:
           name: Lint
           command: yarn lint
   build:
     <<: *defaults
     steps:
-      - attach_workspace:
-          at: ~/project
-      - *restore_node_modules
+      - *attach_workspace
+      - *restore_cache
       - run:
           name: Build distribution
           command: |
-            node -v
+            apt-get -y update && \
+            apt-get install -y --no-install-recommends \
```

Comment:
This should really be in Docker images so we don't need to install the packages on every single build. That'll unnecessarily slow down the build and defeats the purpose of using Docker :/

I can make a Docker image per Node.js version if that'd be useful?

Reply:
We don't need a docker image per node version since this is only needed for the build stage which only happens with Node 8. I don't like having to maintain custom docker images for trivial stuff like this but I'll add back a docker image for this step and update the PR.